modules/observe/adoption-insights/proc-viewing-active-users.adoc

```diff
@@ -0,0 +1,19 @@
+:_mod-docs-content-type: PROCEDURE
+[id="proc-viewing-active-users_{context}"]
+= Viewing the active users
+
+The *Active users* card displays the total number of active users over a specified time range. It also provides a breakdown comparison of *New users* and *Returning users* in a line/area graph format. You can export the user data in a .csv format.
```

Comment:
try "over a specified date and time period." instead. 

"breakdown comparison" is also a bit clunky. You probably should choose just one of those two to aid readability.